### PR TITLE
Refactor Getting Started to the Splash Page

### PR DESCRIPTION
**Pull request by KennethSkylight**
**Overview**

The original getting-started doc was geared towards developers who want to contribute to our repo. Getting started is usually geared to wards developers who want to use our package. Therefore, we are splitting up that doc into contributing.md and a new section in the readme.md page

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,7 +46,66 @@ If you need to base your work on someone else's branch, talk to the branch owner
...
+#### Hardware
+
+Supported hardware for PHDI tools includes:
+* Linux
+* Windows 10/11
+* Mac (Intel or Apple silicone)
+#### Software
+
+##### Overview
+The core development team uses VSCode as its IDE, but other options (e.g. IntelliJ, Eclipse, PyCharm, etc.) can be viable as well. The rest of this document will assume that you're using VSCode as your IDE. The project itself is coded primarily in Python.
```
Comment (spencer-skylight) [inline CONTRIBUTING.md:62]: This strikes me as potentially slightly problematic as the development team changes over time - who knows how future core team members will choose to do development? Also users can see on the main page what languages are used in the code so this is redundant and more prone to becoming out of date (if we decide to add modules or functionality using other languages). Can we change this to:
```
This document will assume that you're using VSCode as your IDE, but other options (e.g. IntelliJ, Eclipse, PyCharm, etc.) can be used as well.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,7 +46,66 @@ If you need to base your work on someone else's branch, talk to the branch owner
...
+```
+
+This should build a single html file in `docs/_build/singlehtml/index.html`.
+
+##### Extensions
+
+The following VSCode extensions help streamline development in the IDE:
+
+**Python**
+The Python extension adds IntelliSense, linting, debugging and other useful tools.
```
Comment (spencer-skylight) [inline CONTRIBUTING.md:108]: Include the following:
```
**Jupyter**
Jupyter extension pack includes renderers and other useful tools for working with Jupyter notebooks.
**autoDocstring**
This extension is helpful to generate docstrings more easily.
```

Also, add a new heading `##### VS Code Settings` with some new content.
The following VS Code settings help enforce team coding conventions.
```
"autoDocstring.docstringFormat": "sphinx-notypes",
"editor.formatOnSave": true,
"editor.rulers": [ 88 ],
"python.formatting.provider": "black",
"python.linting.flake8Enabled": true,
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,34 @@
...
+```
+
+Example import for generic:
+```
+ from phdi.geospatial.smarty import SmartyGeocodeClient
+```
+
+Every building block has a FHIR counterpart that works well with FHIR bundles as inputs. The generic version is used for all other non-FHIR inputs.
+
+For further information on the tutorial: [Geo Spatial Tutorial](tutorials/geospatial-tutorial.md)
```
Comment (spencer-skylight) [inline README.md:45]: Geospatial should be one word.

**Review by spencer-skylight — COMMENTED** [review]
Looks good - just a few suggestions.

> Open an issue on our [issues list](https://github.com/CDCgov/prime-public-health-data-infrastructure/issues) that describes the feature you'd like to see, why you want it, and how it should work. 

The link above should be updated with the new repo name.

File: CONTRIBUTING.md (hunk 1 above)
Comment (KennethSkylight) [reply CONTRIBUTING.md]: Added

File: README.md (hunk 3 above)
Comment (KennethSkylight) [reply README.md]: fixed

**Review by spencer-skylight — APPROVED** [review]
LGTM!

**Pull request merged**